src/app.py

```diff
@@ -255,6 +255,18 @@ def delete_catalogue(uuid: str):
     db.session.commit()
     return jsonify(res)
 
+@app.route("/catalogue/", methods=["DELETE"])
```

Comment:
IMO, the delete endpoint should also have item id to identify which item to delete? ANd then delete all could be delegated to batch delete endpoint perhaps...